dms/migrations/12.0.4.1.0/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    directories = (
+        env["dms.directory"]
+        .sudo()
+        .search(
+            [("storage_id.save_type", "=", "attachment"), ("group_ids", "!=", "[]")]
+        )
+    )
+    if directories:
+        for directory in directories:
+            directory.group_ids = None
```

Comment:
The `if` is not needed:

```suggestion
    for directory in directories:
        directory.group_ids = None
```